src/Core/src/Platform/Windows/ActivityIndicatorExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿#nullable enable
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	public static class ActivityIndicatorExtensions
+	{
+		public static void UpdateIsRunning(this MauiActivityIndicator mauiActivityIndicator, IActivityIndicator activityIndicator)
+		{
+			// TODO: Use IView Opacity if the ActivityIndicator is running.
+			mauiActivityIndicator.ElementOpacity = activityIndicator.IsRunning ? 1 : 0;
+		}
+		public static void UpdateColor(this MauiActivityIndicator mauiActivityIndicator, IActivityIndicator activityIndicator)
+		{
+			mauiActivityIndicator.UpdateColor(activityIndicator, null);
+		}
+
+		public static void UpdateColor(this MauiActivityIndicator mauiActivityIndicator, IActivityIndicator activityIndicator, object? foregroundDefault)
```

Comment:
why is foreground a object?

Reply:
Just because is the type returned by `GetForegroundCache` but, could be a `Brush`.